client/app/bundles/course/material/folders/pages/FolderShow/index.tsx

```diff
@@ -51,15 +52,16 @@ const FolderShow: FC = () => {
   const permissions = useAppSelector(getFolderPermissions);
 
   const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
   useEffect(() => {
-    dispatch(loadFolder(getIdFromUnknown(folderId))).finally(() =>
-      setIsLoading(false),
-    );
+    setIsError(false);
```

Comment:
whenever we navigate to this `FolderShow` page, this whole component will be re-rendered, and hence `isError` will always be re-initialised. Hence, this line should not be necessary

Reply:
When I go to a page that does not exist, `/materials/folders/<non-existent-id>`, then click back on the root folder from breadcrumbs, there is no re-render of the component, and `isError` is not re-initialised.

Reply:
this is strange. I tried this before (and also just now) to click the root folder from breadcrumbs, it loads the entire page again.

However, I see no harm in having this line (since this is just to make sure that the value `isError` is reset) so can just leave it be